python/pyhail/dataset.py

```diff
@@ -830,13 +1081,43 @@ def hardcalls(self):
     def ibd(self, output, maf=None, unbounded=False, min=None, max=None):
         """Compute matrix of identity-by-descent estimations.
 
+        **Examples**
+
+        Suppose we have a variant annotation ``va.panel_maf`` of allele
+        frequencies computed from a reference panel. Then
+
+        >>> (hc.read('example.vds')
+        >>>  .ibd('ibd.tsv', maf='va.panel_maf', min=0.2, max=0.9)
+
+        outputs to ``ibd.tsv`` only those sample pairs with ``pi_hat`` between 0.2 and 0.9 inclusive, using the the provided expression for the minor allele frequency.
+
+        To writes the full IBD matrix to `ibd.tsv` with the minor allele frequency computed from the dataset:
```

Comment:
Be consistent about whether ibd.tsv is code or in italics (single back ticks). Personally, I prefer the italics so that only parameter names or actual code is in code blocks. Not sure what Cotton thinks...